Lombiq.Tests.UI/BasicOrchardFeaturesTesting/WorkflowsFeatureTestingUITestContextExtensions.cs

```diff
@@ -16,13 +16,14 @@ public static class WorkflowsFeatureTestingUITestContextExtensions
             "Test Workflows",
             async () =>
             {
+                var testWorkflowName = "Test workflow";
                 var workflowsPath = "/Workflows/Types";
                 var contentItemPublishTestSuccessMessage = "The content item was published.";
```

Comment:
Shouldn't these be `const`?
```suggestion
                const string testWorkflowName = "Test workflow";
                const string workflowsPath = "/Workflows/Types";
                const string contentItemPublishTestSuccessMessage = "The content item was published.";
```